habitat/core/vector_env.py

```diff
@@ -186,6 +187,9 @@ def _worker_env(
                     else:
                         result = getattr(env, function_name)(*function_args)
                     connection_write_fn(result)
+                elif command == EPISODE_COMMAND:
+                    current_episode = env.current_episode
```

Comment:
This is not needed here, you can directly do `connection_write_fn(env.current_episode)`